### PR TITLE
Fix #92 expand(g:racer_cmd)

### LINKED ISSUE #92: No completion (after update?)
Today I decided to `:PlugUpdate`. Not sure if that's why, but I later today realized no completions at all worked.

It has worked previously, and my `init.vim` file looks like [this](https://github.com/legolord208/.dotfiles/blob/master/neovim/init.vim),
where it's worth to point out
```vim
call plug#begin()
Plug 'rust-lang/rust.vim'
Plug 'racer-rust/vim-racer'
call plug#end()
```

and

```vim
filetype plugin indent on

let g:racer_cmd = '~/.cargo/bin/racer'
let g:racer_experimental_completer = 1
```

No idea why it fails. Still doesn't work if I replace `~` with `/home/legolord208`, so it's not the expansion that is the problem.

EDIT:

> Still doesn't work

*accidentally said "Works" at first*

--------------------------------------------------------------------------

TL;DR `Omni completion (^O^N^P) Pattern not found`

### PR DESCRIPTION
**Pull request by Shougo**

### CODE REVIEW

**Comment by rhysd** [comment]
Looks good. If no other committer cannot see this, I think @Shougo should merge this yourself...

File: autoload/racer.vim [hunk 1]
```diff
@@ -1,9 +1,31 @@
+let s:is_win = has('win32') || has('win64')
+
+function! racer#GetRacerCmd() abort
+  if !exists('g:racer_cmd')
+    let s:sep = s:is_win ? '\' : '/'
+    let s:path = join([
```
Comment (rhysd) [inline autoload/racer.vim]: `s:path` doesn't need to be script local. (`s:sep` and `s:pathsep` are also)

**Review by rhysd — CHANGES_REQUESTED** [review]

File: autoload/racer.vim (hunk 1 above)
Comment (Shougo) [reply autoload/racer.vim]: Ah, OK.  I will fix it.

**Comment by Shougo** [comment]
Merged

**Pull request merged**